doc/nrf/releases/release-notes-1.4.0.rst

```diff
@@ -0,0 +1,1013 @@
+    * Added support for partial reception of RX data (in TCP/IP proxy).
+    * Added AT command ``#XSLMUART`` to change the UART baud rate.
+    * Added data mode support for TCP/UDP proxy client/server.
+    * Added support for the HTTP client service.
+    * Added FOTA support.
+    * Added TLS server support.
+
+* :ref:`mqtt_simple_sample` sample:
+
+    * Added TLS support to the sample.
+
+* :ref:`lib_nrf_cloud` library:
+
+    * Added saving of a valid session flag to settings after all subscriptions have completed, so that the persistent session is only used when the flag is valid.
+    * Replaced ``CONFIG_CLOUD_PERSISTENT_SESSIONS`` usage with Zephyr's :option:`CONFIG_MQTT_CLEAN_SESSION`.
+    * MQTT client ID prefix is now configurable.
+    * Added an option to set a time-out for :c:func:`send()` (:option:`CONFIG_NRF_CLOUD_SEND_TIMEOUT`).
+
+nRF5
+====
+
+The following changes are relevant for the nRF52 and nRF53 Series.
+
+nRF5340 SoC
+-----------
+
+* Added:
+
+  * :ref:`nc_bootloader` sample - This sample demonstrates the usage of an immutable first stage bootloader that has the capability to update the application firmware on the network core of the nRF5340 System on Chip (SoC).
+  * :ref:`nrf_rpc_entropy_nrf53` sample - This sample demonstrates how to use the entropy driver in a dual core device such as nRF5340 PDK.
+  * :ref:`nrfxlib:nrf_rpc` library - This library enables inter-processor communication on Nordic Semiconductor devices.
+    It was added with subpages dedicated to architecture and implementing remote procedure calls.
+
+* Zephyr's :ref:`zephyr:smp_svr_sample` now works on nRF5340 DK.
+
+Bluetooth Low Energy
+--------------------
+
+* Added:
+
+  * :ref:`bms_readme`
+  * :ref:`peripheral_bms` sample - This sample demonstrates how to use the GATT Bond Management Service (BMS).
+  * :ref:`direct_test_mode` sample - This sample demonstrates the Direct Test Mode functions described in Bluetooth Core Specification, Version 5.2, Vol. 6, Part F.
+  * :ref:`gadgets_service_readme` service
+  * :ref:`peripheral_alexa_gadgets` sample - This sample demonstrates how a Bluetooth LE device can connect to an Amazon Echo device using the Alexa Gadgets Bluetooth Service and Profile.
+  * :ref:`bluetooth_central_hr_coded` sample (external contribution) - This sample demonstrates looking for heart rate monitors using LE Coded PHY.
+  * :ref:`peripheral_hr_coded` sample (external contribution) - This sample demonstrates a basic Bluetooth LE Peripheral role functionality that exposes the Heart Rate GATT Service with LE Coded PHY support.
+
+* Updated:
+
+  * Changed default security settings in :ref:`peripheral_hids_keyboard` and :ref:`peripheral_hids_mouse` samples.
+    These samples now require encryption for accessing characteristics.
+  * Added connection attempts filter to :ref:`nrf_bt_scan_readme` library.
+    The filter can be used for blocking peers that disconnected too many times.
+
```

Comment:
New point:

Changed Bluetooth services API naming convention, removing the `_gatt_` and `_c_` infix and using profile names where applicable.